src/misc/setup.cpp

```diff
@@ -1795,3 +1796,11 @@ void Config::ParseArguments()
 
 	arguments.editconf = cmdline->FindRemoveOptionalArgument("editconf");
 }
+
+Config* SETUP_CreateConfig(const int argc, char* argv[])
+{
+	static auto command_line = CommandLine(argc, argv);
+
+	static auto config = Config(&command_line);
+	return &config;
```

Comment:
Aren't you returning a pointer to an object that's about to go out of scope here?

Reply:
Oh, I somehow missed the static keyword.  Ignore me.

Reply:
All good 😎